src/Data/Impact.elm

```diff
@@ -131,12 +143,12 @@ decodeList =
                 |> Pipe.required "short_unit" Decode.string
                 |> Pipe.required "decimals" Decode.int
                 |> Pipe.required "quality" decodeQuality
-                |> Pipe.required "primary" Decode.bool
```

Comment:
Note to reviewers: removed because this field wasn't used anymore in `impacts.json`.